azure-pipelines.yml

```diff
@@ -4,6 +4,7 @@ jobs:
     vmImage: macOS-10.13
   steps:
     - bash: |
+        set -e
```

Comment:
I suggest `set -eu`. You may want to consider `set -eux`.

Reply:
To be consistent with the other PRs will leave as-is for now.